<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnReady">
        <property name="text">
         <string>READY</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnRun">
        <property name="text">
         <string>RUN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>RESET</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnLogging">
        <property name="text">
         <string>LOGGING</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Motor 1</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Current Velocity : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbVel_m1"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbTorqueEnable_m1">
         <property name="text">
          <string>Torque</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current Position : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPos_m1"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnInit_m1">
           <property name="text">
            <string>Initialization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeinit_m1">
           <property name="text">
            <string>Deinitialization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Motor 2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btnInit_m2">
           <property name="text">
            <string>Initialization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeinit_m2">
           <property name="text">
            <string>Deinitialization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Current Velocity : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbVel_m2"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Current Position : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPos_m2"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbTorqueEnable_m2">
         <property name="text">
          <string>Torque</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QTextBrowser" name="tbMessage"/>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Motor 3</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Current Position : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPos_m3"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Current Velocity : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbVel_m3"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="btnInit_m3">
           <property name="text">
            <string>Initialization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeinit_m3">
           <property name="text">
            <string>Deinitialization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbTorqueEnable_m3">
         <property name="text">
          <string>Torque</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Motor 4</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Current Position : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPos_m4"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="btnInit_m4">
           <property name="text">
            <string>Initialization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeinit_m4">
           <property name="text">
            <string>Deinitialization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbTorqueEnable_m4">
         <property name="text">
          <string>Torque</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Current Velocity : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbVel_m4"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Motor 5</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QPushButton" name="btnInit_m5">
           <property name="text">
            <string>Initialization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeinit_m5">
           <property name="text">
            <string>Deinitialization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbTorqueEnable_m5">
         <property name="text">
          <string>Torque</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Current Velocity : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbVel_m5"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Current Position : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPos_m5"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Motor 6</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbTorqueEnable_m6">
         <property name="text">
          <string>Torque</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Current Position : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPos_m6"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Current Velocity : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbVel_m6"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QPushButton" name="btnInit_m6">
           <property name="text">
            <string>Initialization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeinit_m6">
           <property name="text">
            <string>Deinitialization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
